src/Avalonia.Visuals/Platform/IDrawingContextImpl.cs

```diff
@@ -17,6 +17,11 @@ public interface IDrawingContextImpl : IDisposable
         /// </summary>
         Matrix Transform { get; set; }
 
+        /// <summary>
+        /// Gets or sets the base transform of the drawing context.
+        /// </summary>
+        Matrix BaseTransform { get; set; }
```

Comment:
What does "base transform" mean? Why is it different from Transform?

Reply:
its just another tranformation matrix (pre transform/pre translation), that translate all drawings to new position, i didn't want to change all places where Transform is set